rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/SchedulerClient.java

```diff
@@ -1263,6 +1265,25 @@ public UserData getCurrentUserData() throws NotConnectedException {
         }
     }
 
+    @Override
+    public TaskStatesPage getTaskPaginated(String jobId, int offset, int limit)
+            throws NotConnectedException, UnknownJobException, PermissionException {
+        TaskStatesPage taskStatesPage = null;
+        try {
+            List<TaskState> taskStates = restApi().getJobTaskStatesPaginated(sid, jobId, offset, limit)
+                                                  .getList()
+                                                  .stream()
+                                                  .map(taskStateData -> taskState(taskStateData))
+                                                  .collect(Collectors.toList());
+            taskStatesPage = new TaskStatesPage();
+            taskStatesPage.setSize(limit);
```

Comment:
I thought 'limit' was the index of the end of the page, excluded. 
But it seems here that it is the number of tasks in the page. 
Is that correct? Can we make it clearer from the variable name/comment?

Reply:
Actually I have messed up with this limit. Thanks I will try to fix it